realm/realm-library/src/main/java/io/realm/log/RealmLog.java

```diff
@@ -110,6 +110,7 @@ public static void trace(Throwable throwable) {
      * @param message message to log.
      * @param args optional args used to format the message using {@link String#format(String, Object...)}.
      */
+    // called from io_realm_internal_Util.cpp
```

Comment:
Not sure these are necessary. These methods are always called from the Java code.